medusa/providers/torrent/html/gftracker.py

```diff
@@ -175,14 +174,15 @@ def process_column_header(td):
 
                     torrent_size = cells[labels.index('Size/Snatched')].get_text(strip=True).split('/', 1)[0]
                     size = convert_size(torrent_size, units=units) or -1
-
+                    pubdate = cells[labels.index('Added')].get_text(" ")
+                    
```

Comment:
But it worked as it and got shown correctly in the manual page,  dates in python aren't  always objects? So that they can be manipulated